librz/asm/arch/evm/evmasm.c

```diff
@@ -0,0 +1,359 @@
+// SPDX-FileCopyrightText: 2023 gogo <gogo246475@gmail.com>
+// SPDX-FileCopyrightText: 2012-2020 pancake <pancake@nopcode.org>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_util.h>
+#include <rz_types.h>
+#include <rz_asm.h>
+#include <string.h>
+#include <stdlib.h>
+
+static int evmAsm(RzAsm *a, RzAsmOp *op, const char *buf) {
+	int j, len = 1;
+	size_t i;
+	if (!a || !op || !buf) {
+		return 0;
+	}
+	ut8 opbuf[40] = { 0 };
+	
+	if (strcmp("stop", buf) == 0) {
```

Comment:
convert all these `if (strcmp(XXXXX, buf) == 0) {` to `if (!strcmp(XXXXX, buf)) {`